src/Field/WfsSearch/WfsSearch.jsx

```diff
@@ -50,6 +50,11 @@ export class WfsSearch extends React.Component {
      * @type {Object}
      */
     searchAttributes: PropTypes.object.isRequired,
+    /**
+     * An object mapping feature types to an array of attribute details.
+     * @type {Object}
+     */
+    attributeDetails: PropTypes.object,
```

Comment:
Please make use of `PropTypes.shape`.
This way you can define the existing detail keys and their types.
e.g.:
```javascript
// An object taking on a particular shape
  attributeDetails: PropTypes.shape({
    exactSearch: PropTypes.bool,
    type: PropTypes.oneOf(['int', 'string', 'number', …]),
    …
  })
```

Reply:
Alright, added in 3dec246.